build/includes/minikube.mk

```diff
@@ -39,9 +39,14 @@ minikube-shell: $(ensure-build-image)
 # via `make build` or `make build-images` into the "agones" minikube instance.
 minikube-push:
 	$(MINIKUBE) image load $(sidecar_linux_amd64_tag) -p $(MINIKUBE_PROFILE)
-	$(MINIKUBE) image load $(controller_tag) -p $(MINIKUBE_PROFILE)
-	$(MINIKUBE) image load $(ping_tag) -p $(MINIKUBE_PROFILE)
-	$(MINIKUBE) image load $(allocator_tag) -p $(MINIKUBE_PROFILE)
+	$(MINIKUBE) image load $(controller_amd64_tag) -p $(MINIKUBE_PROFILE)
+	$(MINIKUBE) image load $(ping_amd64_tag) -p $(MINIKUBE_PROFILE)
+	$(MINIKUBE) image load $(allocator_amd64_tag) -p $(MINIKUBE_PROFILE)
+
+	$(MINIKUBE) image tag $(sidecar_linux_amd64_tag) $(sidecar_tag) -p $(MINIKUBE_PROFILE)
```

Comment:
As you noted, this seems likely to break if minikube is running on an ARM machine (e.g. an M1 mac).

Reply:
I've sent https://github.com/googleforgames/agones/pull/2663 to update the docs so that this knowledge is more easily discoverable.